.github/workflows/yocto-build-deploy.yml

```diff
@@ -213,6 +212,28 @@ jobs:
         shell: bash --noprofile --norc -eo pipefail -x {0}
 
     steps:
+      # Generate an app installation token that has access to
+      # all repos where the app is installed (usually the whole org)
+      # Owner input to make token valid for all repositories in the org
+      # This behvaiour is required for private submodules
+      - name: Generate GitHub App installation token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+        id: app_token
+        with:
+          app_id: ${{ vars.FLOWZONE_APP_ID }}
+          installation_retrieval_mode: organization
+          installation_retrieval_payload: ${{ github.repository_owner }}
+          private_key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+          # actions:read - We are fetching workflow run results of a merge commit when workflow is triggered by new tag, to see if tests pass
+          # pull-requests:write - # Read is required to fetch the PR that merged, in order to get the test results. Write is required to create PR comments for workflow approvals.
+          # conents:read - required for cloning private submodules
+          permissions: |-
+            {
+              "contents": "read",
+              "actions": "read", 
```

Comment:
The Flowzone app does not have `actions: read` IIRC, and neither does the new defaults for the automatic GITHUB_TOKEN. I think we need to decide which is better to use here.